src/sentry/static/sentry/app/views/eventsV2/eventView.tsx

```diff
@@ -391,7 +391,7 @@ class EventView {
       id: undefined,
       start: undefined,
       end: undefined,
-      statsPeriod: undefined,
+      statsPeriod: eventViewV1.statsPeriod,
```

Comment:
Should we avoid using eventViewV1 as much as possible? I'm assuming that we want to deprecate it at some point.

Reply:
@leedongwei good point. The `EventViewv1` type was something that was around during a refactor. At this point, it's a name that's no longer relevant.

But as a type, it's still useful to be used a "template". Mayhaps the type name `EventViewTemplate` may be better.